lib/colorado_lottery.rb

```diff
@@ -0,0 +1,32 @@
+require "./lib/contestant"
+require "./lib/game"
+require "pry"
+
+class ColoradoLottery
+  attr_reader :registered_contestants, :winners
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+  end
+
+  def current_contestants
+    @registered_contestants
+  end
+
+  def interested_and_18?(contestant, game)
+    contestant.game_interests.include?(game.name) && contestant.age >= 18
+  end
+
+  def can_register?(contestant, game)
+    interested_and_18?(contestant, game) && (game.national_drawing? || contestant.out_of_state? == false)
```

Comment:
`contestant.out_of_state? == false` is redundant, because `out_of_state` will always return either a true of false value, so we would never need to ask `true == false` or `false == false`